app/model/entity/Group.php

```diff
@@ -107,6 +112,65 @@ public function statsArePublic(): bool {
     return $this->publicStats;
   }
 
+  /**
+   * @ORM\Column(type="boolean")
+   */
+  protected $isArchived;
```

Comment:
Is not this a bit overkill? isArchived is equivalent to (archivationDate != NULL).